src/plugins/translate/plugin.translate.js

```diff
@@ -127,27 +154,71 @@ export class TranslatePlugin extends BookReaderPlugin {
         translatedParagraph.setAttribute('data-translate-index', `${pageIndex}-${pidx}`);
         translatedParagraph.className = 'BRparagraphElement';
         const originalParagraphStyle = paragraphs[pidx];
-  
+        
+        const lineHeightAvg = [];
+        let paragraphSpans = Array.from(originalParagraphStyle.childNodes).filter((ele) => {
+          return ele.tagName === 'SPAN';
+        })
+        for (const ele of paragraphSpans) {
+          lineHeightAvg.push(parseInt(ele.style["line-height"]));
+        }
+        const lineHeightCalc = lineHeightAvg.reduce((acc, curr) => acc + curr, 0) / lineHeightAvg.length;
+        const fontSize = `${parseInt($(originalParagraphStyle).css("font-size"))}px`;
+
         $(translatedParagraph).css({
           "margin-left": $(originalParagraphStyle).css("margin-left"),
           "margin-top": $(originalParagraphStyle).css("margin-top"),
           "top": $(originalParagraphStyle).css("top"),
           "height": $(originalParagraphStyle).css("height"),
-          "font-size": `${parseInt($(originalParagraphStyle).css("font-size")) - 3}px`,
           "width": $(originalParagraphStyle).css("width"),
+          "font-size": fontSize,
+          "line-height": `${lineHeightCalc}px`,
+        });
+
+        translatedParagraph.addEventListener('mousedown', (e) => {
```

Comment:
Let's drop the hover effect for now in light of our discussion with Jim; we will switch to the floating button later.